### ISSUE TITLE
BobRadio Integration

### ISSUE THREAD

**Issue by bayu01** [GuardsmanPanda/GManBot#1]
Design and Implement a basic API that will interact with the chat, to rate songs, @GuardsmanPanda add/remove songs?

**Comment by GuardsmanPanda** [comment]
add/remove? I am not sure I follow.
Do you meana way for peopel to leave song requests?

**Comment by bayu01** [comment]
yeah so is the playlist a fixed set of songs?  or how do you envision adding new songs or removing songs, regardless of the integration with the twitch chat.   So say, a new album came out from your favorite singer.  You'll need a way to add them to the mix, whether it is from the UI, or programatically.

**Comment by GuardsmanPanda** [comment]
The gmanbot is not playlist aware, when a song is played it simply adds the
ratings and quote to a database under the name of the current song.

The bobapp/website/radio should probably take a winamp exported playlist as
input and have song interface for connecting an entry to a YouTube video

**Comment by GuardsmanPanda** [comment]
each playlist import should then compare against the current playlist and remove / ask removal confirmation for songs not found
